### DISCUSSION TITLE
how to calibrate phase field parameters when coupling with crystal plasticity

### DISCUSSION THREAD

**Discussion by RWTHLHK** [idaholab/moose#29023] [Q&A Modules: Phase field]
I combined phase field fracture with crystal plasticity. But the damage results look weird. When the stress_yy is 2,8GPa, the damage value is almost 0. But I used the parameters from phase_field-fracture test/tests, the results shouldn't seem so different.
![image](https://github.com/user-attachments/assets/3c1e26a9-3571-4bb8-adde-aa125548a262)
stress results:
![image](https://github.com/user-attachments/assets/a9611576-1b50-4082-9dc0-e358775ecceb)
damage results:
![image](https://github.com/user-attachments/assets/66a4775b-9c0b-4eb9-90bb-17d2362e3431)

**Comment by GiudGiud** [comment]
Hello

These results do not look converged in the spatial resolution of the mehs.
Please use uniform refinement to see if the results imrpove

Guillaume

**Reply by RWTHLHK** [reply]
I created a finer mesh but the damage still doesn't increase:
![image](https://github.com/user-attachments/assets/b6f7beda-9fd5-4fcc-a25d-82010b0f14a5)
![image](https://github.com/user-attachments/assets/bd3a325f-0ae6-49ea-bab4-f3936975a3ef)
I used default values for CP part and phase field parameters from test files. The order of parameters magnitude should be the same.

**Reply by GiudGiud** [reply]
@dewenyushu do you have insights on this?

**Reply by RWTHLHK** [reply]
i found the reason. In material computeCrackedStress, only 2 strain types are supported: mechanical_strain and elastic_strain. But for CP model, the strain is named as total_lagranrian_strain. So the strain computeCrackedStress is always 0.
